.github/CODEOWNERS

```diff
@@ -25,15 +25,15 @@
 /src/lib/util/Pokedex.ts @favna
 
 # QuantumlyTangled
-# Now while I may not have made all of them I would like to be notified of them so... this is what the following line is for
-/docker/ @quantumlytangled
 /src/commands/GameIntegration/ffxiv.ts @quantumlytangled
 /src/commands/Twitch/ @quantumlytangled
-/src/events/twitchStreamOffline.ts @quantumlytangled
-/src/events/twitchStreamOnline.ts @quantumlytangled
+/src/events/twitch/ @quantumlytangled
+/src/events/analytics/ @quantumlytangled
+/src/lib/types/Enums.d.ts @quantumlytangled
```

Comment:
Are you sure you want this? You're going to get GH pinged for things that do not particularly concern analytics.